### DISCUSSION TITLE
T1000-E Tracker for Meshtastic | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1631] [Q&A]
# T1000-E Tracker for Meshtastic | Seeed Studio Wiki

SenseCAP Card Tracker T1000-E for Meshtastic

https://wiki.seeedstudio.com/sensecap_t1000_e/

**Comment by Coopersmith-24601** [comment]
In Meshtastic, how do I monitor the light sensor?

I assume I have to turn on the "Detection Sensor" module, but which GPIO pin would I monitor?

**Reply by Salamimonster** [reply]
You cant.
See the current state [here](https://forum.seeedstudio.com/t/i-need-help-with-the-sensecap-card-track-t1000-e/281101/11?u=weatherman).
